src/pineko/check.py

```diff
@@ -66,6 +66,29 @@ def check_grid_and_eko_compatible(pineappl_grid, operators, xif):
     if not np.all(in1d(np.unique(operators["targetgrid"]), np.array(x_grid))):
         raise ValueError("x grid in pineappl grid and eko operator are NOT compatible!")
 
+def is_fonll_b(fns, lumi):
+    """Checks if the fktable we are computing is a DIS FONLL-B fktable
+
+    Parameters
+    ----------
+        fns : str
+            flavor number scheme (from the theory card)
+        lumi : list(list(tuple))
+            luminosity info
+
+    Returns
+    -------
+            : bool
+            true if the fktable is a FONLL-B DIS fktable
+    """
+    isDIS = True
+    for lists in lumi:
+        for el in lists:
+            if el[1] != 11:
+                isDIS = False
```

Comment:
Instead of defining `isDIS` (even wrong case convention), just early return

Reply:
```suggestion
                return False
```

Reply:
Of course remove `isDIS` even above and below, and add a comment explaining why you are early returning.